### PR TITLE
commiting with pageObjectModel logging

### PR DESCRIPTION
**Pull request by tipuranjalid**
Please review the code

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -36,6 +36,33 @@
...
+			<groupId>org.apache.logging.log4j</groupId>
+			<artifactId>log4j-core</artifactId>
+			<version>3.0.0-alpha1</version>
+		</dependency>
+
+		<dependency>
+			<groupId>org.apache.logging.log4j</groupId>
+			<artifactId>log4j-to-slf4j</artifactId>
+			<version>2.8.2</version>
+		</dependency>
```
Comment (Ravichandra-C) [inline pom.xml]: Why did you add this dependency?

File: src/test/java/com/testCases/BaseTest.java [hunk 2]
```diff
@@ -11,8 +13,13 @@
 
 public class BaseTest {
 	public WebDriver driver;
+	public Logger logger;
 	@BeforeClass
-	void setUp() {
+	public void setUp() {
+		System.setProperty("log4j2.xml","C:\\Users\\ctipuran\\eclipse-workspace\\MavenProject\\src\\test\\resources\\log4j2.xml");
```
Comment (Ravichandra-C) [inline src/test/java/com/testCases/BaseTest.java]: Why is the property name log4j2.xml, Is that correct? where did you get the property name from?

**Comment by Ravichandra-C** [comment]
Got it working in my local, Added the comments, Fix them

**Review by Ravichandra-C — CHANGES_REQUESTED** [review]

File: src/test/java/com/testCases/BaseTest.java (hunk 2 above)
Comment (Ravichandra-C) [reply src/test/java/com/testCases/BaseTest.java]: Read this for the correct key name [Automatic configuration](https://logging.apache.org/log4j/2.x/manual/configuration.html#automatic-configuration)

Comment (Ravichandra-C) [reply src/test/java/com/testCases/BaseTest.java]: Also change the path to relative path

Comment (tipuranjalid) [reply src/test/java/com/testCases/BaseTest.java]: its working when log4j2.xml is located in classpath
